chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -78,10 +78,15 @@ developer.contact.emailRequired=Developer contact email address is required.
 developer.contact.phoneRequired=Developer contact phone number is required.
 developer.contact.nameRequired=Developer contact name is required.
 developer.websiteRequired=Developer website is required.
-developer.websiteIsInvalid=Developer's website is improperly formed.
+developer.websiteIsInvalid=Developer website is improperly formed.
 developer.status.noCurrent=The developer must have a current status specified.
 developer.status.duplicateStatus=The status '%s' cannot be listed twice in a row.
 
+#system-specific developer errors
+system.developer.transparencyAttestationIsNullOrEmpty=The developer transparency attestation in the system is null or empty.
+system.developer.transparencyAttestationNotMatching=A transparency attestation does not exist in the system where the ACB name matches the pending certifying body name.
```

Comment:
All public facing mentions of ACBs need to refer to them as the full "ONC-ACB". Similar with "ONC-ATL"